packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -115,7 +116,7 @@ export default class ContentSwitcher extends React.Component {
               ...data,
               index: this.state.selectedIndex,
               name: child.props.name,
-              text: child.props.text,
+              text: <Text>child.props.text</Text>,
```

Comment:
I think this would be a breaking change since we're changing the type of `this.props.onChange`. I don't know if we need to wrap this in `Text` either since this isn't a user-facing snippet of text